src/index.html

```diff
@@ -10,19 +10,21 @@
       <h1>What's Cookin'</h1>
     </header>
     <nav>
-      <button type="button" name="User Name" id="userName">User Name</button>
+      <h2 id="userName"></h2>
+      <button type="button" name="All Recipes" id="allRecipesBtn">All Recipes</button>
       <button type="button" name="Favorite Recipes" id="favoriteRecipes">Favorite Recipes</button>
       <button type="button" name="What to Cook" id="whatToCook">What to Cook</button>
     </nav>
     <main>
       <section class="search-box">
         <form class="search-options">
-          <input type="text" name="search" placeholder="Search Recipe by Name or Indgredient" id="searchBar">
```

Comment:
Went with the easiest solution that we could think of!  It involved creating new search buttons to hide and show on the DOM based on what was being searched (all recipes, favorite recipes).